README.md

```diff
@@ -1,32 +1,32 @@
 # Deploy Cognite Function action
-This action deploys a Python function to Cognite Functions with schedule defined.
+This action deploys a Python function to Cognite Functions with a schedule defined.
 
 ## Inputs
 ### Function metadata in Github Workflow
 #### Required
 1. `function_name`: Name of your function. That name will become an `external_id` and *must* be unique within your project
-2. `function_folder`: Parent folder of function code, defaults to `.` (i.e. the root of your repo)
+2. `function_folder`: Parent folder of function code defaults to `.` (i.e. the root of your repo)
 3. `cdf_deployment_credentials`: Name of Github secrets that holds the API-key that shall be used to deploy the function. 
-That API-key should have following CDF capabilities: `Files:READ`, `Files:WRITE`, `Functions:READ`, `Functions:WRITE`
+That API-key should have the following CDF capabilities: `Files:READ`, `Files:WRITE`, `Functions:READ`, `Functions:WRITE`
```

Comment:
Arrrr11!!!

Reply:
The THE-ifying has begun... @scottmelhop